src/account/components/AccountLogin.js

```diff
@@ -33,10 +33,9 @@ import PageHeader from 'layout/PageHeader';
 import PageLoader from 'layout/PageLoader';
 import LocalePicker from 'localization/components/LocalePicker';
 import { useAnalytics } from 'monitoring/analytics';
+import { fetchAuthURLs } from 'state/account/accountSlice';
```

Comment:
yes, this is all code that i will want to move (or at least most of what is in each file). i realize the naming isn't going to make the most sense right now, but didn't want to put in the effort to come up with a better folder structure that we know we're going to rearrange again in a couple weeks. i guess it is sort of a `model` folder now rather than `state`.

Reply:
`model` is still confusing, I think this is related to the api interaction in general right? I think `terrasoApi` or something related to the api might make sense for now